CMakeLists.txt

```diff
@@ -45,6 +46,7 @@ IF(NODEJS)
   FIND_PACKAGE(NodeJS)
 
   ADD_NODEJS_MODULE(${PROJECT_NAME} src/nodejs_bindings.cpp)
+  ADD_NODEJS_MODULE(${PROJECT_NAME} src/lookup_bindings.cpp)
```

Comment:
I don't understand this - should this be a single

    ADD_NODEJS_MODULE(${PROJECT_NAME} src/nodejs_bindings.cpp src/lookup_bindings.cpp)

? Not sure how node-cmake behaves if we create two node modules with the same name.